### DISCUSSION TITLE
A way to disable access log?

### DISCUSSION THREAD

**Discussion by andreymal** [dani-garcia/vaultwarden#1451] [General]
I use nginx as a reverse proxy which has its own logs, so bitwarden_rs access log like `POST /identity/connect/token (login) => 200 OK` is just an unnecessary duplicate which I want to disable. 

I tried `ROCKET_LOG=critical` and `EXTENDED_LOGGING=false` but access log is still present.

`LOG_LEVEL=warn` is inappropriate because it hides too many other logs which I don't want to hide (like `logged in successfully` etc.)

Is there a way to disable only access log without touching any other logs? If not, treat this topic as a feature request.

**Comment by BlackDex** [comment]
What is your end goal? Those items are important in case of issues between nginx > bitwarden for example.
Also, the flow of message could be useful in case of issues.

The only way you can do what you want would be to change the code of bitwarden it self, but i think that would be too much trouble for the worth of it.

**Reply by andreymal** [reply]
>What is your end goal?

Remove duplicate logs, reduce used disk space, remove visual garbage without using grep or something similar.

>Those items are important in case of issues between nginx > bitwarden for example.

I have no issues (in fact, I've never had any nginx-related issues in the last ten years), so I want to disable it; these logs are not important for me. Even if issues do happen, I can always just enable the log :)

The flow is already available in /var/log/nginx/access.log in my case.

**Reply by BlackDex** [reply]
Well, they arn't in your case, but maybe they are for others. And i think this is a very small corner case and for only disabling those requests we would need to create a whole new environment variable to just disable that, which in my opinion is too much to add.

You could always enable syslog and filter those messages using the syslog capabilities.
